pkg/liquidity-mining/contracts/gauges/StakelessGauge.sol

```diff
@@ -32,6 +34,7 @@ abstract contract StakelessGauge is IStakelessGauge, ReentrancyGuard {
     event Checkpoint(uint256 indexed periodTime, uint256 periodEmissions);
 
     // solhint-disable var-name-mixedcase
+    uint256 public max_relative_weight;
```

Comment:
Make this a private variable (e.g. `_relativeWeightCap`) and provide a getter to it instead

Reply:
Done.